packages/eslint-mdx/src/types.ts

```diff
@@ -37,10 +37,10 @@ export interface LocationError {
 }
 
 export interface ParserOptions extends Linter.ParserOptions {
-  extensions?: string | string[]
-  markdownExtensions?: string | string[]
+  extensions?: string[] | string
+  markdownExtensions?: string[] | string
   filePath?: string
-  parser?: string | ParserConfig | ParserFn
+  parser?: ParserConfig | ParserFn | string
```

Comment:
interesting, was order causing an issue? Or was this something a linter suggested?

Reply:
It's fixed by rule [@typescript-eslint/sort-type-union-intersection-members](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/sort-type-union-intersection-members.md).

Reply:
good to know, thanks!